doc/reference/provided_metrics.md

```diff
@@ -154,6 +158,28 @@ The following internal metrics are provided:
   - Number of active warnings
 ```
 
+(api-rates-metrics)=
+## API rates metrics
+
+The API rates metrics include the `lxd_api_requests_completed_total` and `lxd_api_requests_ongoing`. These metrics can be consumed by an observability tool deployed externally (either Canonical COS or third parties) to help identify failures or overload on a LXD server. Thresholds for these metrics' values can be set on these tools to trigger alarms and take programatic actions.
+These metrics consider all endpoints in the [LXD REST API](api.md), plus the internal endpoints. They also include a label `entity_type` based on the main entity type that the endpoint is operating on. This label can have one of the following values:
+- `instance`
+- `image`, for endpoints that operate on images or image aliases.
+- `storage_pool`, for endpoints that operate on storage pools, storage volumes, storage backups or storage buckets.
+- `network`, for endpoits that operate on networks, network zones or network ACLs.
```

Comment:
```suggestion
- `image`, for endpoints that operate on images or image aliases
- `storage_pool`, for endpoints that operate on storage pools, storage volumes, storage backups or storage buckets
- `network`, for endpoints that operate on networks, network zones or network ACLs
```

Reply:
The same change below - we should be consistent with punctuation within a list. So either all items get a full stop at the end, or none of them do. (And in this case none makes way more sense. :) )